src/IQFeed.CSharpApiClient/Streaming/Level1/ILevel1EventCommon.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using IQFeed.CSharpApiClient.Streaming.Common.Messages;
+using IQFeed.CSharpApiClient.Streaming.Level1.Messages;
+
+namespace IQFeed.CSharpApiClient.Streaming.Level1
+{
+    public interface ILevel1EventCommon
```

Comment:
IBaseLevel1Event maybe... I dont know. Its minor. Lets review that later.
Lets merge this baby!